avocado/utils/memory.py

```diff
@@ -320,3 +320,57 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+class MemInfo(object):
+    """
+    Representation of /proc/meminfo
+    """
+
+    class MemInfoItem(object):
+        """
+        Representation of one item from /proc/meminfo
+        """
+        def __init__(self, name, safe_name):
+            self.name = name
+            self.safe_name = safe_name
```

Comment:
Just for convenience, given it's the name used to create the MemInfo attribute. We can drop it safely for sure, if you want.